examples/typescript/yarn.lock

```diff
@@ -10082,6 +10073,14 @@ react-virtuoso@^1.5.8:
     react-app-polyfill "^1.0.6"
     resize-observer-polyfill "^1.5.1"
 
+react-virtuoso@^2.2.0:
```

Comment:
having two react-virtuosos in yarn.lock does not seem right

Reply:
oh interesting, that second one might be coming from the MML library, let me check...

Reply:
yep it's coming from `mml-react` dependency

Reply:
should we make it a peer dep in that repo?

Reply:
I would rather upgrade it TBH

Reply:
ok i'll upgrade it in `mml-react`